### ISSUE TITLE
Broken reference in package.json

### ISSUE THREAD

**Issue by Ahrengot** [maisano/react-router-transition#1]
[Package.json points to `lib/index.js`](https://github.com/maisano/react-router-transition/blob/master/package.json#L5) but the generated file is called `react-router-transition.js`. This naturally causes a bug in Webpack at compile time:

```
ERROR in ./path/to/file.js
Module not found: Error: Cannot resolve module 'react-router-transition' in /path/to/project
 @ ./path/to/file.js 13:29-63
```

I wanted to patch this and submit a pull request, but both my attempts at fixing this didn't work. First I tried changing the reference to `main` in package.json so it pointed at `react-router-transition.js` instead, then, when that didn't work, I renamed the compiled files filename to `index.js`. Both solutions netted the same result: Webpack just imports an empty object that looks like this when inspected in the browser: `{ default: {} }`. I'm not sure why renaming the file doesn't work, but there's something funky going on here ...

**Comment by maisano** [comment]
d'oh! i used the incorrect file for the `main` property and forgot to add a `libraryTarget` property to the webpack config. thanks for pointing this out.

**Comment by maisano** [comment]
gonna close this for now, as it should be patched as of `0.0.4`–thanks again!

**Comment by Ahrengot** [comment]
Sweet! Thanks for the quick turnaraound :+1: :)
